.travis.yml

```diff
@@ -10,6 +10,10 @@ before_script:
   - cp keys/saml.key.enc.example keys/saml.key.enc
   - bin/rake db:setup --trace
   - bin/rake assets:precompile
+after_script:
+  - npm install -g pa11y-crawl
+  - bundle exec rails s -d
+  - pa11y-crawl --standard Section508 --ci http://localhost:3000
```

Comment:
I'm OK with a travis run script and/or including pa11y-crawl in our packages.json. 

The main reason for including in the after_script is to allow faster failures and we're still in discovery-mode to see if pa11y through continua11y is useful. pa11y-crawl doesn't even return failure status if there are errors; we need to confirm that pa11y failures warrant a hold on promoting a PR.